packages/flutter_goldens/lib/flutter_goldens.dart

```diff
@@ -451,8 +451,11 @@ class FlutterLocalFileComparator extends FlutterGoldenFileComparator with LocalC
 }
 
 /// A [FlutterGoldenFileComparator] for skipping golden image tests when the
-/// current environment is not supported. This comparator is used in post-submit
-/// checks on LUCI.
+/// current environment is not supported.
+///
+/// The [FlutterLocalFileComparator] is a catch-all for testing environments not
+/// covered by the other [FlutterGoldenFileComparator]s. This comparator is used
```

Comment:
Is the "This" supposed to refer to the Local comperator mentioned in the previous sentence or to the class that's documented here?